physics/rigid_motion.hpp

```diff
@@ -28,19 +28,42 @@ using geometry::RigidTransformation;
 using geometry::Vector;
 using geometry::Velocity;
 using quantities::Acceleration;
+using quantities::AngularAcceleration;
 using quantities::Length;
 using quantities::Variation;
 using quantities::si::Radian;
 
+// A trait to determine if Frame is FromFrame or ToFrame and return the other
+// one as the |type| member.
+template<typename Frame, typename FromFrame, typename ToFrame>
+struct other_frame;
+
+template<typename Frame>
+struct other_frame<Frame, Frame, Frame> {
```

Comment:
Do we need this one? `ToFrame` = `FromFrame` makes `angular_velocity_of<Frame>` rather confusing.

Reply:
We need this one.  There are tests that compute `r * r.Inverse()` and check the result.  Without this specialization the other two become ambiguous.